security.txt

```diff
@@ -0,0 +1,26 @@
+-----BEGIN PGP SIGNED MESSAGE-----
+Hash: SHA256
+
+# If you have any queries, please address them to the team leader, Jack Orcherton. His PGP key can be found below & submit your query on the contact form or reach out to him anyway on his personal website https://jackorcherton.github.io/.
```

Comment:
Shameless plug much? x3 Just a small grammatical fix too -- gonna have to resign the doc ofc
```suggestion
# If you have any queries, please address them to the team leader, Jack Orcherton. His PGP key can be found below & submit your query on the contact form or reach out to him using the details provided [here](https://jackorcherton.github.io/).
```

Reply:
Ok you caught me 🤣